scripts/markdown-table.sh

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env bash
+
+# Script copied from: https://josh.fail/2022/pure-bash-markdown-table-generator/
```

Comment:
Note this whole script to generate a markdown table copied from https://josh.fail/2022/pure-bash-markdown-table-generator/